cmake/thirdparty/FindMFEM.cmake

```diff
@@ -19,18 +19,15 @@ endif()
 message(STATUS "Looking for MFEM using MFEM_DIR = ${MFEM_DIR}")
 serac_assert_is_directory(VARIABLE_NAME MFEM_DIR)
 
-set(_mfem_cmake_config "${MFEM_DIR}/MFEMConfig.cmake")
+set(_MFEM_DIR ${MFEM_DIR}) # Save MFEM_DIR as a non-cache variable
+find_package(MFEM CONFIG NO_DEFAULT_PATH PATHS "${MFEM_DIR}/lib/cmake/mfem")
+# find_package will overwrite MFEM_DIR, so restore it here
```

Comment:
why does their find_package overwrite MFEM_DIR?  What does it get overwritten to?

Reply:
I think it's the default behavior of `find_package` - if the config is found it'll be overwritten with the directory containing the `MFEMConfig.cmake`, otherwise it'll be set to `NOTFOUND`.